### PR TITLE
*: move crd definition out of operator.yaml into crd.yaml

### PR DESCRIPTION
**Pull request by fanminshi**
fixes https://github.com/operator-framework/operator-sdk/issues/76

cc/ @hasbro17 @spahl

### CODE REVIEW

**Comment by fanminshi** [comment]
Manual Test:
```sh
$ operator-sdk new app-operator --api-version=app.example.com/v1alpha1 --kind=App
$ cd  app-operator

$ cat deploy/crd.yaml
apiVersion: apiextensions.k8s.io/v1beta1
kind: CustomResourceDefinition
metadata:
  name: apps.app.example.com
spec:
  group: app.example.com
  names:
    kind: App
    listKind: AppList
    plural: apps
    singular: app
  scope: Namespaced
  version: v1alpha1

$ cat deploy/operator.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: app-operator
spec:
  replicas: 1
  selector:
    matchLabels:
      name: app-operator
  template:
    metadata:
      labels:
        name: app-operator
    spec:
      containers:
        - name: app-operator
          image: quay.io/fanminshi/example:app-operator
          command:
          - app-operator
          imagePullPolicy: Always

$ kubectl create -f deploy/rbac.yaml
role "app-operator" created
$ kubectl create -f deploy/crd.yaml
customresourcedefinition "apps.app.example.com" created

$ operator-sdk build quay.io/fanminshi/example:app-operator
$ docker push quay.io/fanminshi/example:app-operator
The push refers to a repository [quay.io/fanminshi/example]

$ kubectl create -f deploy/operator.yaml
deployment "app-operator" created
$ kubectl create -f deploy/cr.yaml
app "example" created

$ kubectl get po
NAME                            READY     STATUS    RESTARTS   AGE
app-operator-867fb9f787-bh8nf   1/1       Running   0          8m
busy-box                        1/1       Running   0          7m
```

File: pkg/generator/gen_deploy.go [hunk 1]
```diff
@@ -22,40 +22,55 @@ import (
...
 	KindPlural   string
 	GroupName    string
 	Version      string
-	ProjectName  string
-	Image        string
+}
+
+// renderCRDYaml generates deploy/crd.yaml
+func renderCRDYaml(w io.Writer, kind, apiVersion string) error {
+	t := template.New(crTmplName)
```
Comment (hasbro17) [inline pkg/generator/gen_deploy.go]: I know the template name doesn't actually matter but this should be `crdTmplName`.

Comment (fanminshi) [reply pkg/generator/gen_deploy.go]: @hasbro17 yeah, the name doens't matter. just for formality i guess. good catch on this one.

**Comment by hasbro17** [comment]
LGTM

**Pull request merged**